picard/ui/options/cover.py

```diff
@@ -54,8 +54,7 @@ class CoverOptionsPage(OptionsPage):
         config.BoolOption("setting", "save_images_overwrite", False),
         config.ListOption("setting", "ca_providers", [
             ('Cover Art Archive', True),
-            ('Amazon', True),
```

Comment:
Was this forgotten in https://github.com/metabrainz/picard/pull/1101?

Reply:
Yes, that's a left over from this change.